Basic Rules.txt

```diff
@@ -350,6 +350,47 @@ Using a medikit:
 Medic-kit, non-medic user: works as if you are a medic with skill 30.
 Medic-kit, medic user: Works as normal, but add 30 to amount healed.
 
+
+==============================
+Death, Dying and Downed
+==============================
+Oh no! You have less than 1 health point left! What now!?!
+
+The good news is you are not* dead yet. You are bleeding out. 
+When your character falls below 1 hit point, you get 3 saving throws called Death 
+Saving Throws. These throws are not tied to any attribute. When you fail any three 
+times, you die. If you succeed, you do not lose one of these three Death 
+Saving Throws, but will still have to roll next round if you are not stabalized. 
+Once you have expended a saving throw, the next time you fall 
+below 1 health point your saving throws fall to 2. the next time, you have 1. you 
+can only replenish these saving throws by going a whole encounter without having 
+to use any. Each encounter that you go without expending a saving throw 
+replenishes one saving throw up to a maximum of three.

```

Comment:
An example of intended mechanic:
Player falls to -4 hit points. On their next turn, they roll a 23, failing the saving throw. Next turn, they roll a 17, bad luck. Luckily, before their next turn a comrade gets to them and uses a medical kit to patch them up. $ turns later, the character walks on a landmine and is once again downed. Previously, they had 3 fail opportunities and used two. This time, they have 2 fail opportunities. On their fnext turn, they roll a 48, failing the throw. On the turn after, they roll an 89 and are safe for the round. Another buddy gets to them and stabilizes them. The encounter ends.

Later that day, the group is ambushed, and our unfortunate player is ganked. On their next turn, they roll a 92 and are safe. But before a friend gets to them in time, they roll a 10 on their next turn. They are now dead.

Lets rewind to before the ambush. Our example player decides instead of leading the group into the dark mine, they are going to provide rear cover. In the ambush, they are not downed, and after the encounter ends, they are still standing. (They have now gone one full encounter without being downed). The enxt day, a brawl breaks out in the restaraunt, one thing leads to another, and the player is enveloped by a plasma grenade, reducing their hit points to < 0. They have regained one Death Saving Throw after the ambush from the previous day. On their first turn, they roll a 2, failing the DC. The turn after that, they roll a 37, also failing the DC. They are now dead.

Reply:
Sounds great! :)

Reply:
Makes sense to me. This change is supposed to be tracked in the death and dying rules change in PR #155 though. This PR is specifically about the added feat.

Reply:
lolol I didn't notice that we were on a different PR, woops